cli/__snapshots__/install_spec.js

```diff
@@ -260,8 +260,6 @@ Installing Cypress (version: 1.2.3)
 
 ⚠ Warning: It looks like you've installed Cypress globally.
 
-  This will work, but it's not recommended.
-
   The recommended way to install Cypress is as a devDependency per project.
 
   You should probably run these commands:
```

Comment:
Maybe get rid of the probably here?
```suggestion
  You should run these commands:
```

Reply:
yeah, I'm not sure that we KNOW they globally installed it. I almost removed this as well.... 🤔

Reply:
Ahh yeah. The wording does make it seem like we do not know for sure